cdap-cli/src/main/java/co/cask/cdap/cli/command/ExecuteQueryCommand.java

```diff
@@ -47,7 +47,7 @@
  */
 public class ExecuteQueryCommand extends AbstractAuthCommand {
 
-  private static final long TIMEOUT_MS = 30000;
+  private static final long TIMEOUT_SEC = 3600;
```

Comment:
Ya thats a good idea. Will update the PR to take in an optional timeout argument (in mins).